docs/proposals/20200804-windows-support.md

```diff
@@ -0,0 +1,271 @@
+    - [Goals](#goals)
+    - [Non-Goals/Future Work](#non-goalsfuture-work)
+  - [Proposal](#proposal)
+    - [Cluster API Bootstrap Provider Kubeadm](#cluster-api-bootstrap-provider-kubeadm)
+      - [cloud-init and cloudbase-init](#cloud-init-and-cloudbase-init)
+      - [VHD Creation](#vhd-creation)
+      - [Kubelet and other component configuration](#kubelet-and-other-component-configuration)
+      - [netbios names](#netbios-names)
+    - [Infrastructure provider implementation](#infrastructure-provider-implementation)
+    - [User Stories](#user-stories)
+      - [As an operator, I would like to create Windows OS worker nodes with the CAPI API.](#as-an-operator-i-would-like-to-create-windows-os-worker-nodes-with-the-capi-api)
+      - [As an operator, I would like to manage Windows OS worker nodes with the CAPI API.](#as-an-operator-i-would-like-to-manage-windows-os-worker-nodes-with-the-capi-api)
+    - [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
+      - [Signing of the components.](#signing-of-the-components)
+      - [Known prototypes and prior work:](#known-prototypes-and-prior-work)
+    - [Security Model](#security-model)
+    - [Risks and Mitigations](#risks-and-mitigations)
+  - [Alternatives](#alternatives)
+  - [Upgrade Strategy](#upgrade-strategy)
+  - [Additional Details](#additional-details)
+    - [Test Plan [optional]](#test-plan-optional)
+    - [Graduation Criteria [optional]](#graduation-criteria-optional)
+      - [Alpha](#alpha)
+      - [Beta](#beta)
+      - [Stable](#stable)
+    - [Version Skew Strategy](#version-skew-strategy)
+  - [Implementation History](#implementation-history)
+
+## Glossary
+
+Refer to the [Cluster API Book Glossary](https://cluster-api.sigs.k8s.io/reference/glossary.html).
+
+If this proposal adds new terms, or defines some, make the changes to the book's glossary when in PR stage.
+
+## Summary
+
+This proposal is for the support of the Windows nodes with CAPI and [infrastructure providers](https://cluster-api.sigs.k8s.io/reference/glossary.html#infrastructure-provider) that wish to support Windows [OS](https://cluster-api.sigs.k8s.io/reference/glossary.html#operating-system) worker nodes.  
+Windows has been Stable in Kubernetes since 1.14 and is supported in mix worker node scenarios.  
+
+Windows support will provide the ability to add Windows nodes to a [workload cluster](https://cluster-api.sigs.k8s.io/reference/glossary.html#workload-cluster).  Windows nodes are configured differently across 
+solutions and the centralization of the scripts and guidance in the Cluster API solution will provide 
+guidance for the community on best practices for configuring Windows nodes at scale.
+
+Windows node support has some unique challenges because of the current limitations of Windows Containers. 
+Windows containers do not have privileged containers which means that configuration and access to the host
+machine must be done at provisioning time.  
```

Comment:
Nit: Maybe reword to either 'Windows containers do not support privileged operations' or 'Windows does not support privileged containers'.
Windows containers do not have privileged containers is a bit confusing to me.